circuitClient.js

```diff
@@ -122,6 +124,68 @@ class CircuitClient {
       }]);
     });
   }
+  
+  async setMyStatusMessage(statusMessage) {
+    return this.client.setStatusMessage(statusMessage)
+    .catch(console.error);
+  }
+
+  getMyStatusMessage() {
```

Comment:
I think you don't need to create  this, you can do the same as this.getPresence since assntially you are just returning the api.